sonar-project.properties

```diff
@@ -1,16 +1,48 @@
-# must be unique in a given SonarQube instance
-sonar.projectKey=inspec_inspec_AYahEXtgJ4YHsO5MtHEr
+# SonarQube configuration file
+#
+# Sample Ruby Sonar file - https://docs.sonarsource.com/sonarqube-server/10.6/analyzing-source-code/languages/ruby/
+# properties defined in https://docs.sonarsource.com/sonarqube-server/latest/analyzing-source-code/analysis-parameters/
+#
+# view this repo in sonar: <ADD your URL like https://sonar.progress.com/dashboard?id=chef_...3b9c01e2&codeScope=overall>
+# Required metadata
+sonar.projectKey=inspec_inspec_6028d24d-5d03-4bd1-97f5-b64a44b02ca5
```

Comment:
Why do we need to use new `projectKey` and new `projectName`? These projects are already confused in sonar cloud.